specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/virtualNetworkGateway.json

```diff
@@ -2459,6 +2459,24 @@
       ],
       "description": "VPN client revoked certificate of virtual network gateway."
     },
+    "RadiusServer": {
+      "properties": {
+        "radiusServerAddress": {
```

Comment:
>"radiusServerAddress" [](start = 8, length = 21)

if any of these properties are required when specifying a radius server please mark them as such

Reply:
Done for radius address